man/memtier.7

```diff
@@ -3,14 +3,19 @@
 .\"
 .TH "MEMTIER" 7 "2021-03-01" "Intel Corporation" "MEMTIER" \" -*- nroff -*-
 .SH "NAME"
-libmemtier.so \- TODO
+libmemtier.so - internal library which enables memory tiering
```

Comment:
Is "memory tiering" the established term for this?

Reply:
@michalbiesek , @bratpiorka , @jschmieg - memkind memory tiering? what do you think?